backend/Testing/ApiTests/UsersICanSeeQueryTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Text.Json.Nodes;
+using Shouldly;
+using Testing.Services;
+
+namespace Testing.ApiTests;
+
+[Trait("Category", "Integration")]
+public class UsersICanSeeQueryTests : ApiTestBase
+{
+    private async Task<JsonObject> QueryUsersICanSee(bool expectGqlError = false)
+    {
+        var json = await ExecuteGql(
+            $$"""
+              query {
+                usersICanSee(take: 10) {
+                  totalCount
+                  items {
+                    id
+                    name
+                  }
+                }
+              }
+              """,
+            expectGqlError, expectSuccessCode: false);
```

Comment:
Yes, that was leftover from my first attempt. But this class is going to be deleted anyway since this was the integration test suite that I've replaced with unit tests. (Which need a real database so they're not *exactly* "unit" tests, but close enough).